### PR TITLE
Replace usage of removed db.indexes() with SHOW INDEXES

### PR DESCRIPTION
**Pull request by burqen**

### CODE REVIEW

**Review by Lojjs — APPROVED** [review]

File: core/src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java [hunk 1]
```diff
@@ -229,7 +229,7 @@ private void exportSchema(PrintWriter out) {
     }
 
     private List<String> exportIndexes() {
-        return db.executeTransactionally("CALL db.indexes()", Collections.emptyMap(), result -> result.stream()
+        return db.executeTransactionally("SHOW INDEXES", Collections.emptyMap(), result -> result.stream()
```
Comment (jexp) [inline core/src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java:232]: can we presume that the result columns have not changed between these two calls?

Comment (burqen) [reply core/src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java:232]: Nope, there is one change: "provider" is replaced by "indexProvider"

Comment (burqen) [reply core/src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java:232]: But from what I could see in this method we never care about that field so I think we are good.

**Comment by burqen** [comment]
Amended some updates. Ping @Lojjs

**Comment by burqen** [comment]
I will merge this and monitor https://live.neo4j-build.io/buildConfiguration/Apoc_Dev_BuildAndCreateArtifacts?mode=builds#all-projects

**Comment by burqen** [comment]
... well... shit... I can't merge either...

**Comment by Lojjs** [comment]
[APOC failures due to removal of db.indexes() and db.constraints()](https://trello.com/c/KJUMznPU/2751-apoc-failures-due-to-removal-of-dbindexes-and-dbconstraints)

**Pull request merged**